harmony-kotlin/src/commonMain/kotlin/com.harmony.kotlin/data/repository/CacheRepository.kt

```diff
@@ -30,16 +30,17 @@ class CacheRepository<V>(
     return when (operation) {
       is DefaultOperation -> get(query, CacheSyncOperation())
       is MainOperation -> getMain.get(query)
-      is CacheOperation -> getCache.get(query).let {
-        try {
-          if (!validator.isValid(it)) {
+      is CacheOperation -> {
+        val cacheValue = getCache.get(query)
+        return try {
```

Comment:
Just noticed that this "try" should be wrapping also the getChage.get(query) as it could throw an exception.

This is not a huge problem but it is worth changing it for two reasons:
- We are not receiving that possible exception on the fallback lambda and we should
- CacheSyncOperation wrap the cache call inside the try/catch (so this one should behave the same)

It is not something related to your changes so I am approving the PR. We must change this on this PR or in the following one fixing the CacheSyncOperation fallback

Reply:
Fix applied to #60